src/components/Game.js

```diff
@@ -0,0 +1,43 @@
+import { useMemo, useState } from "react";
+import Board from "./Board";
+import calculateWinner from "./Winner";
+
+function Game(props) {
+  const [xIsNext, setxIsNext] = useState(true);
+  const [box, setBox] = useState(Array(9).fill(""));
```

Comment:
Optional: Maybe we could define`Array(9).fill("")` as a constant variable since we also call it when we want to reset the board

Reply:
in case you want to improve performance you could use [lazy-initial-state](https://reactjs.org/docs/hooks-reference.html#lazy-initial-state)